modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -348,14 +288,179 @@ processed.
 As said, you can wrap that command in a Cron job so it’s run every 5
 minutes for example.
 
+=== WND Listener Configuration
+
+Create a service following the instructions below that checks the share for changes:
```

Comment:
I think we should remark this is for systemd. This is recommended (if systemd is being used) but technically not needed.

Reply:
remarked